client/src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { gql } from "apollo-boost";
+import { graphql } from "react-apollo";
+
+const fetchRecipes = gql`
```

Comment:
Added graphql integration and a sample query. Will work when it is merged into `development`